Sources/Views/MessagesCollectionView.swift

```diff
@@ -67,5 +70,31 @@ open class MessagesCollectionView: UICollectionView {
         guard let indexPath = indexPathForLastItem else { return }
         scrollToItem(at: indexPath, at: .bottom, animated: animated)
     }
+    
+    override open func reloadData() {
```

Comment:
@azurechen Thanks for submitting this PR! Unfortunately, I don't think overriding `reloadData` is the right approach to achieving this functionality. 

I notice this is the second PR you've submitted recently. You should join the [MessageKit Slack channel](https://join.slack.com/t/messagekit/shared_invite/enQtMjI4NzIzNzMyMzU0LTEyNmYzNzBlNjNiYjAwYjY1ZjhkMmVjNDE4OGNlMDY1OTRmNzVmMzI4MTg3OWQzZTM2NGU4NTU3ZDE5NzkwNzQ) so we can discuss design decisions. 

I think some MessageKit users have already implemented this functionality on their own, it would be good to discuss/compare it to what they've done.